.npmrc

```
@@ -1 +1,3 @@
 save-exact=true
+; Set a new registry for a scoped package
+@testing-library:registry=https://registry.npmjs.org/
```

We'd want to always use the public npm registry for all packages
```suggestion
registry=https://registry.npmjs.com
```